tests/functional/settings_clients.js

```diff
@@ -178,9 +181,33 @@ define([
         .end()
 
         // clicking disconnect on the second device should update the list
-        .findByCssSelector('.client:nth-child(2) .client-disconnect')
-          .click()
-        .end()
+        .then(click('.client:nth-child(2) .client-disconnect'))
+
+        // get the modal dialog
+        .then(testElementExists('.intro'))
+        .then(testElementExists('.disabled'))
+
+        // test cancel
+        .then(click('.cancel-disconnect'))
+        .then(FunctionalHelpers.pollUntil(function () {
```

Comment:
It looks like you are waiting for client-disconnect to go away. I've always looked for a reason to use leadfoot's [waitForDeletedByCssSelector](https://theintern.github.io/leadfoot/module-leadfoot_Command.html#waitForDeletedByCssSelector). Valid use?